ui/webui/src/components/AnacondaWizard.jsx

```diff
@@ -345,6 +348,14 @@ const Footer = ({
                                         {_("To continue, select the devices to install to.")}
                                     </HelperTextItem>
                                 </HelperText>}
+                            {activeStep.id === "installation-review" && reviewWarning &&
```

Comment:
This `&& reviewWarning` is redundant - it's always true. But does not hurt.

Reply:
Yes, it should be always true but though being a bit conscious could not hurt